elfi/methods/bo/acquisition.py

```diff
@@ -244,6 +246,170 @@ def evaluate_gradient(self, x, t=None):
         return grad_mean - 0.5 * grad_var * np.sqrt(self._beta(t) / var)
 
 
+class MaxVar(AcquisitionBase):
+    r"""The maximum variance acquisition method.
+
+    The next evaluation point is acquired in the maximiser of the variance of
+    the unnormalised approximate posterior.
+
+    \theta_{t+1} = arg max Var(p(\theta) * p_a(\theta)),
+
+    where the unnormalised likelihood p_a is defined
+    using the CDF of normal distribution, \Phi, as follows:
+
+    p_a(\theta) =
+        (\Phi((\epsilon - \mu_{1:t}(\theta)) / \sqrt(\sigma_{1:t}(\theta) + \sigma_n))),
+
```

Comment:
Now that I look this again, unlike I commented before, I think you should use sigma^2 here, otherwise the user might be confused as sigma usually refers to standard deviation and not variance as here! (Sigma is used for the variance in the code though but it is better to keep the potential confusion there, or if you can change that, that would be of course good)

Reply:
I think sigma in code should always refer to standard deviation. Please rename to sigma2 or var or something else if it's variance.

Reply:
Yeah, I think this would be the safest and best way to go

Reply:
Renamed to `sigma2_n`.